packages/flutter_tools/lib/executable.dart

```diff
@@ -39,7 +39,7 @@ import 'src/runner/flutter_command.dart';
 /// Main entry point for commands.
 ///
 /// This function is intended to be used from the `flutter` command line tool.
-Future<Null> main(List<String> args) async {
+Future<void> main(List<String> args) async {
```

Comment:
This can just be `void` now (no need for `Future<void>`.

Reply:
This brings up a lint: `[dart] avoid async functions that return void`